packages/pretty-proptypes/src/renderPropType.js

```diff
@@ -5,6 +5,7 @@ import convert, { getKind, reduceToObj } from 'kind2string';
 import allComponents from './components';
 
 const IGNORE_COMMENTS_STARTING_WITH = ['eslint-disable', '@ts-'];
+const DEPRECATE_PROP_THAT_CONTAIN = '@deprecated';
```

Comment:
Super nit: this should be "DEPRECATE_PROPS_THAT_CONTAIN" to be grammatically correct

Reply:
Changed! Yesss it was fun 😄